src/js/tracks/text-track.js

```diff
@@ -172,6 +172,8 @@ class TextTrack extends Track {
     this.cues_ = [];
     this.activeCues_ = [];
 
+    this.preload_ = !!this.tech_.preloadTextTracks;
```

Comment:
Looks like the issue is that this ends up defaulting to `false`, at least during the tests since `preloadTextTracks` ends up not being set for tests.
It might be better to just move the line from above here to make sure that we default to true.
```
this.preload_ = this.tech_.options_.preloadTextTracks !== false;
```
However, it would mean updating some tests to make sure that `options_` is present in the tests. Specifically, the tests towards the bottom of this file (https://github.com/videojs/video.js/blob/master/test/unit/tracks/text-track.test.js#L534) which use the `testTech` variable. Since it's created via `EventTarget`, it needs to just get a `testTech.options_ = {}` before passing it into the `TextTrack` constructor.

Reply:
I guess leaving the line to set `this.preloadTextTracks` in tech.js is fine, we'll just need to set it with `!== false` here as well. This would mean the least amount of changes overall.

Reply:
@brion pinging myself a reminder to clean this up tonight/tomorrow.

Reply:
Ok, applying the second recommendation (using a !== false in this bit and keeping the option on the tech)